src/main/java/org/gridsuite/loadflow/server/service/LoadflowObserver.java

```diff
@@ -19,9 +19,12 @@ public class LoadflowObserver {
 
     private final ObservationRegistry observationRegistry;
 
-    private static final String OBSERVATION_PREFIX = "app.loadflow.";
+    private static final String OBSERVATION_PREFIX = "app.";
     private static final String PROVIDER_TAG_NAME = "provider";
 
+    private static final String COMPUTATION_NAME_TAG_NAME = "computation.name";
+    private static final String COMPUTATION_NAME = "loadflow";
```

Comment:
What about "type=loadflow" ? or computation-type, for labels I think you don't have to use "." as a separator